WrightTools/artists/_quick.py

```diff
@@ -228,13 +231,14 @@ def quick2D(
     chopped = data.chop(xaxis, yaxis, at=at, verbose=False)
     # colormap
     # get colormap
-    if data.channels[channel_index].signed:
-        cmap = "signed"
-    else:
-        cmap = "default"
-    cmap = colormaps[cmap]
-    cmap.set_bad([0.75] * 3, 1.0)
-    cmap.set_under([0.75] * 3, 1.0)
+    if cmap is None:
+        if data.channels[channel_index].signed:
+            cmap = "signed"
+        else:
+            cmap = "default"
+        cmap = colormaps[cmap]
+        cmap.set_bad([0.75] * 3, 1.0)
+        cmap.set_under([0.75] * 3, 1.0)
```

Comment:
It's probably best practice to consolidate this code block with `_parse_cmap` in the artists base module.  But the decisions in this code block are actually different than that parser, so some decisions need to be made before we do it.  

In any case, this change is not needed for this PR.  Just saying to keep it in mind.